tools/export/uvision/__init__.py

```diff
@@ -290,13 +290,19 @@ def build(project_name, log_name='build_log.txt', cleanup=True):
         out, err = p.communicate()
         ret_code = p.returncode
 
+        # Print logs if something goes wrong
+        if not os.path.exists(log_name) or ret_code != 0:
+            print("Build failed! \nOut:{}\n\nErr:{}".format(out.decode(), err.decode()))
```

Comment:
The last exporter run reports this: http://mbed-os.s3-eu-west-1.amazonaws.com/builds/exporter/8301/PASS/DISCO_L476VG/uvision5/f16e55c8e4b7f4d70fa22b7a6c1236b67837b0d0_exporter_build_log_DISCO_L476VG_uvision5.txt

Means we get here. log name might not exists (that is why we have seen the tracebacks earlier ?), or ret_code is nonzero? Should we actually handle two separately, or what is the suggestion? The entire build passes, I checked the entire log.

Reply:
I might have run CI too early, let me know, I can stop the build asap

Reply:
If log file does not exists or ret code is something else than zero tells that something went wrong..

I am not sure should the script stop here with error or not. It seems that there is better logic in upper level which is aiming to run all exporters and tell some conclusion. I assumed that we want to use it

Reply:
The print is

```
Building mbed-os-example-blinky DISCO_L476VG uvision5
Build failed! 
Out:

Err:
*** Using Compiler 'V5.06 update 3 (build 300)',
```

`out` and `error` are empty ?

Reply:
interesting